specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -88,7 +95,7 @@
               }
             },
             "schema": {
-              "$ref": "#/definitions/GroupQuotasEntity"
+              "$ref": "#/definitions/LROResponse"
```

Comment:
>LROResponse [](http://example.com/codeflow?start=37&length=11)

Why doesn't it return GroupQuotasEntity and provisioningState?

Reply:
I think, it was changed based on the comments in PR: https://github.com/Azure/azure-rest-api-specs-pr/pull/15893 
201 responses should have provisioningState.
The response has 
"properties": {
          _"provisioningState": "Accepted"_
        }